foundations/introduction/join_the_odin_community.md

```diff
@@ -39,7 +39,7 @@ When asking your question, please remember to include the context:
 
 If you can't pinpoint the problem, you can share a **screenshot**. This is especially useful for showing the output of commands in the command line. In Discord, drag and drop your screenshot image file into the chat box to upload it or simply use the PrtScn and paste keyboard shortcuts. If you don't know how to take a screenshot on your computer, this is a good time to ask Google.
 
-If the puzzle still can't be solved, push your project to GitHub or put your code in a [replit](https://replit.com) so that others can comb through and debug the code. You'll learn all about GitHub very soon.
+If you do share a screenshot of the output of your code, make sure to push your project to GitHub or put your corresponding code in a [replit](https://replit.com) so that others can comb through and debug the code. The screenshot of the output and the correlating code will help make it easier to understand the problem for people helping you. You'll learn all about GitHub very soon.
```

Comment:
I think the wording here is a little confusing about when is a good time to use a screenshot. The aim of this section should be to make clear that when seeking help, the person should use all of the tools at their disposal to give anyone helping them enough information. That may include a screenshot for things like terminal outputs or visual design problems but it will often need corresponding code file(s) and if possible code that is runnable and reproduces the problem.

Reply:
I updated it with the situations where a screenshot could be useful. I can't think of any other situations where a screenshot would be useful. I did include it to state to always include the corresponding files with the respective error. I think it will help make it clearer when to use the screenshot.